app/views/posts/_write.haml

```diff
@@ -79,4 +83,4 @@
     #post-editor.padding-10= yield :form
     .subber.centered= yield :buttons
   - else
-    = render 'editor', item: post
+    = render 'editor', item: post.replies.first
```

Comment:
`post.written`*, for consistency? Plus this might not inherently order `post.replies` so we may have issues with arbitrary database ordering.

Reply:
I think that failed on preview somehow

Reply:
Can we solve that in the controller, by forcibly setting `post.written` if it's not already set there? That way it avoids some of the other pitfalls that this might here.